manifests/08_cr.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operator.openshift.io/v1
+kind: "ClusterCSIDriver"
+metadata:
+  name: "ebs.csi.aws.com"
+spec:
+  driverConfig:
+    driverName: "ebs.csi.aws.com"
+  logLevel: Debug
+  managementState: Managed
+  operatorLogLevel: Debug
```

Comment:
did we want debug in here?

Reply:
nope, just until I get the operator working :)

Reply:
fixed